### PR TITLE
Available locales table in docs

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

Added available locales table in storybook docs

# Why

Give our users information on which locales are supported.

# How Tested

Verified on storybook

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by alkwa-msft** [comment]
What the locales table looks like
image

File: packages/storybook/stories/Localization.stories.mdx [hunk 1]
```diff
@@ -40,11 +40,91 @@ For the example below, we will localize buttons in a `ControlBar` in French (Fra
 
 ### Composites
 
-If you want to localize your composite, assign a `ComponentLocale` to the `locale` prop of the composite. You may import one of our locales like `COMPONENT_LOCALE_FR_FR` for French (France)
-in the example below.
+If you want to localize your composite, assign a `ComponentLocale` to the `locale` prop of the composite. You may import one of our composite locales like
```
Comment (anjulgarg) [inline packages/storybook/stories/Localization.stories.mdx:43]: I don't understand this line. Might be easier to understand with a code sample.

**Review by JamesBurnside — APPROVED** [review]

File: packages/storybook/stories/Localization.stories.mdx (hunk 1 above)
Comment (mgamis-msft) [reply packages/storybook/stories/Localization.stories.mdx]: Simplified the wording on the description a bit. But there is an sample after the description, although it is quite lengthy.

File: packages/storybook/stories/Localization.stories.mdx [hunk 2]
```diff
@@ -40,11 +40,91 @@ For the example below, we will localize buttons in a `ControlBar` in French (Fra
...
-If you want to localize your composite, assign a `ComponentLocale` to the `locale` prop of the composite. You may import one of our locales like `COMPONENT_LOCALE_FR_FR` for French (France)
-in the example below.
+If you want to localize your `ChatComposite` or `CallComposite`, import one of our composite locales such as `COMPOSITE_LOCALE_FR_FR` for French (France) and assign it
+to the `locale` prop like in the example below.
 
 <Source code={LocalizedComposites} />
 
+## Available locales
+
+The following is a table of our available locales for components and composites which can be imported from our package. The component strings are passed as
```
Comment (prprabhu-ms) [inline packages/storybook/stories/Localization.stories.mdx:50]: nit: table of locales

File: packages/storybook/stories/Localization.stories.mdx [hunk 3]
```diff
@@ -40,11 +40,91 @@ For the example below, we will localize buttons in a `ControlBar` in French (Fra
...
+    <td>COMPONENT_LOCALE_ZH_CN</td>
+  </tr>
+  <tr>
+    <td>Chinese (Taiwan)</td>
+    <td>COMPONENT_LOCALE_ZH_TW</td>
+    <td>COMPONENT_LOCALE_ZH_TW</td>
+  </tr>
+</table>
+
+Note: More locales will be supported in future releases.
```
Comment (prprabhu-ms) [inline packages/storybook/stories/Localization.stories.mdx:126]: I wouldn't add this. Let's not promise stuff, but deliver it regardless :)

**Review by prprabhu-ms — APPROVED** [review]

**Review by alcail — APPROVED** [review]

**Review by anjulgarg — APPROVED** [review]

**Pull request merged**